akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/PathDirectivesSpec.scala

```diff
@@ -464,6 +464,75 @@ class PathDirectivesSpec extends RoutingSpec with Inside {
     }
   }
 
+  "the `ignoreTrailingSlash` directive" should {
+    var passes = 0
+    val route = ignoreTrailingSlash {
+      passes = passes + 1
```

Comment:
Yes, this is the weakest thing, it also doesn't support parallel tests nicely I think (because of the tests checks). I will try to find a better way to count the passes (`AtomicInteger` looks like a good candidate)